stdlib/2and3/threading.pyi

```diff
@@ -63,14 +65,14 @@ class Thread:
     else:
         def __init__(self, group: None = ...,
                      target: Optional[Callable[..., Any]] = ...,
-                     name: Optional[str] = ...,
+                     name: Optional[_String] = ...,
```

Comment:
You can just use `typing.Text` in argument types; it implicitly includes str.

Reply:
@JelleZijlstra Okay - I noticed that `_String = Union[str, unicode]` was defined in `urlparse.pyi` (prior to this PR); is there a reason for that?

Reply:
In return types it may make a bit more of a difference, or it may just be unnecessary.

Reply:
> In return types it may make a bit more of a difference

Can you clarify what you mean by this?

Reply:
`Text` is equivalent to `Union[str, unicode]` in argument types, but in other places (like return types) it is equivalent to just `unicode`. (For Python 2.)

Reply:
Thanks for the clarification, corrected in 7ba0cc02.